packages/yoroi-extension/chrome/extension/background.js

```diff
@@ -1614,6 +1658,28 @@ function handleInjectorConnect(port) {
   });
 }
 
+function metadataIsValid(policyId, assetName, metadatas): any {
+  let metadataWrapper = null;
+  for (const m of metadatas) {
+    if (m.key === '721') {
+      metadataWrapper = m;
+    }
+  }
+  if (metadataWrapper === null) {
+    return null;
+  }
+  const metadata = metadataWrapper.metadata;
+  if (metadata.version && metadata.version !== '1.0') {
+    return null;
+  }
+  const asset = metadata[policyId]?.[assetName];
+  if (!asset) {
+    return null;
+  }
```

Comment:
You can combine both of these two checks into one if you wish! 

```js 
  if (
    (metadata.version && metadata.version !== '1.0') ||
    !metadata[policyId]?.[assetName]
  ) {
    return null;
  }
```

Reply:
don't want to do that because they actually check for different things